exercises/G-numbers/exercise.js

```diff
@@ -1 +1,6 @@
 // Start by creating a variables `numberOfStudents` and `numberOfMentors`
+numberOfStudents = 15;
+numberOfMentors = 8;
+total = numberOfMentors + numberOfStudents
```

Comment:
Variables in JavaScript need to be "declared"; this means you prefix the variable name with `let` or `const` (e.g. `const numberOfStudents = x`, `let numberOfMentors = y`. This is effectively letting JS know that `numberOfStudents` refers to a variable, instead of being some other key word.